elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/SubCollectionPageTotalsQueryBuilder.java

```diff
@@ -104,7 +104,7 @@ public Query build() {
             predicates.stream().forEach(predicate -> {
                 predicate.getPath().add(0, new FilterPredicate.PathElement(
                         parentType,
-                        parentType.getSimpleName(),
+                        dictionary.getEntityFor(parentType),
```

Comment:
There are lots of other places that build predicates.  We should be consistent and use this new method everywhere we build a FilterPredicate.PathElement OR a new Path.